thirdeye-server/src/main/java/ai/startree/thirdeye/PluginLoader.java

```diff
@@ -77,13 +79,15 @@ public PluginLoader(
       final ContributorsFinderRunner contributorsFinderRunner,
       final BootstrapResourcesRegistry bootstrapResourcesRegistry,
       final PostProcessorRegistry postProcessorRegistry,
-      final PluginLoaderConfiguration config) {
+      final PluginLoaderConfiguration config,
```

Comment:
nit: let's make `PluginLoaderConfiguration`, the first or the last dependency.

Reply:
fyi. This is purely cosmetic. was thinking it would be more organized to have the config and then all the registries or vice versa. However, having the config somewhere in the middle would be a bit iffy